### ISSUE TITLE
Preserve column widths when changing class or adding a property to a class in the browser

### ISSUE THREAD

**Issue by bmunkholm** [realm/realm-studio#588]
When a new property is added to a Class, all columns are resized to their default size.

Ver 1.14.0

**Comment by RetVal** [comment]
Any updates? This issue make the user experience really bad.

**Comment by kraenhansen** [comment]
@RetVal no particular update here. @bmunkholm / @roberhofer should we bump this up on the backlog?

**Comment by bmunkholm** [comment]
I agree this is annoying - it is however just something that happens while adding properties and there is a very clear workaround. I believe there are more annoying and important updates to do first.

@RetVal We would obviously LOVE to accept a PR should you get too annoyed by this :-)
